lib/cartopy/mpl/gridliner.py

```diff
@@ -121,45 +157,69 @@ def __init__(self, axes, crs, draw_labels=False, xlocator=None,
             to determine the locations of the gridlines in the y-coordinate of
             the given CRS. Defaults to None, which implies automatic locating
             of the gridlines.
+        xformatter: optional
+            A :class:`matplotlib.ticker.Formatter` instance to format
+            longitude labels.
+        yformatter: optional
+            A :class:`matplotlib.ticker.Formatter` instance to format
+            latitude labels.
```

Comment:
These descriptions seems a little misleading when we consider L.202 and L.205, which cause the default values to become `LongitudeFormatter` and `LatitudeFormatter` rather than None.  I'd like these docstrings to reflect that behaviour please.

Reply:
Fixed